### ISSUE TITLE
Roll-up your sleeves - lets apply your Weka knowledge !

### ISSUE THREAD

**Issue by damirsaleschoice** [Sales-Choice-Volunteering-Project/EmotionAnalyzerWeka#10]
We will need to work together on this.

I will supply a dataset with parsed sentences with a desired output, and you will find the best-working algorithm.
It will be a CSV file with either a bunch of words, or a bunch of numbers with one word as a result.

First, we will try words-only, and if results are bad, then we will do the numbers.
I will explain what the numbers are and what they mean.

You may also need to learn the attribute selection in Weka, but we will talk about it if necessary.

This ticket will be updated as we progress with out work. Please complete all other Weka tickets first !

**Comment by sherlockliang888** [comment]
I tried with NB and SVM (linear, rbf, sigmoid) on finance new data (4800+ instances), which are multiclass labeled {positive, negative, neutral}, NB and SVM with linear give the best result after a 80% split, with an accuracy of 72% and 75% respectively. It is not great but it is good based on what we have. I guess we will go with SVM linear for now.

**Comment by damirsaleschoice** [comment]
Yes we could. There is also a feature called "Auto Weka" which chooses the best algorithms automatically. Try installing it the same way you did the forecasting, and see if it works.

**Comment by sherlockliang888** [comment]
Let me try that real quick.
